### DISCUSSION TITLE
Signer vs. Signer V2

### DISCUSSION THREAD

**Discussion by ihedbit** [Layr-Labs/eigensdk-rs#180] [General]
I noticed that the EigenSDK-Go contains two signer modules. Are there any differences between them, or have they been combined into a single signer module?

**Comment by MegaRedHand** [comment]
Hello! Sorry for the late response!

The signer was deprecated from the Go SDK and signer v2 was the newest implementation. Since the Rust SDK was newer it didn't have this problem and had the signer v2 functionality from the get-go.
